exercises/060_floats.zig

```diff
@@ -23,15 +23,12 @@
 //     const pi: f16 = 3.1415926535;   // rounds to 3.140625
 //     const av: f16 = 6.02214076e+23; // Avogadro's inf(inity)!
 //
-// A float literal has a decimal point. When performing math
+// A float literal doesn't need a decimal point. When performing math
 // operations with numeric literals, ensure the types match. Zig
 // does not perform unsafe type coercions behind your back:
```

Comment:
I like your change.  Now I'm realizing the part about "math operations" doesn't really follow the first sentence or even reflect how Zig works anymore. Maybe split this into two paragraphs?

```
// A float literal can have a decimal point, but it doesn't require one.
//
// // When dealing with numeric literals, ensure the types match. Zig does not
// perform unsafe type coercions behind your back:
```

Reply:
Mentioning that float literals don't need a decimal point is just necessary because other languages need one. I think we can leave it implicit with an example.